websites/K/keybr.com/presence.ts

```diff
@@ -0,0 +1,165 @@
+const presence: Presence = new Presence({
+  clientId: '812689086435164251'
+});
+
+const startTime: number = new Date().getTime();
+let presenceData: PresenceData = {
+  startTimestamp: startTime,
+  largeImageKey: 'logo'
+};
+
+presence.on('UpdateData', async (): Promise<void> => {
+  if (window.location.pathname === '/') {
+    const settings: {
+      [ key: string ]: string
+    } = {
+      details: await presence.getSetting('changedetails'),
+      state: await presence.getSetting('changestate')
+    };
+    const _log = console.log;
+    console.log = () => {};
+    console.log(settings);
+    console.log = _log;
```

Comment:
!?!?!?!?

Reply:
Settings does not update (for me) without the log line. Callum said that the whole thing is broken for him